Therapeut Vechter/Assets/Scripts/LevelScreen/GameObjectSpawn.cs

```diff
@@ -6,25 +6,31 @@ namespace LevelScreen
 {
     public class GameObjectSpawn : MonoBehaviour
     {
-        public GameObject LevelPrefab;
-        public LevelData[] LevelData;
+        [SerializeField]private GameObject levelPrefab;
+        [SerializeField]private LevelData[] levelData;
         public int SpawnDistance;
         public int ObjectsSpawned;
         private Transform[] objList;
+        
         //get the last 2 objects of the spawned in levels
         private int lastIndexInt;
-        public GameObject lastArrayObj;
+        private GameObject lastArrayObj;
         private int secondLastIndexInt;
-        public GameObject secondLastArrayObj;
+        private GameObject secondLastArrayObj;
 
         private void Start()
         {
-            for (var i = 0; i < LevelData.Length; i++)
+            for (var i = 0; i < levelData.Length; i++)
             {
-                var newLevel  = Instantiate(LevelPrefab, new Vector3(150+ i*SpawnDistance,0,0),Quaternion.identity,transform);
-                newLevel.GetComponent<LevelScript>().LoadLevelData(LevelData[i]);
+                var newLevel  = Instantiate(levelPrefab, new Vector3(275+ i*SpawnDistance,0,0),Quaternion.identity,transform);

```

Comment:
275 is a magic number, assign a private variable to the number and replace 275 with that.
Example: private float canvasXOffset = 275; and then replace the 275 with the variable